### PR TITLE
Team component

### PR DESCRIPTION
**Pull request by Tch4lla**
Added links to socials  for those who provided them 

Added Roles

### CODE REVIEW

File: components/Team/TeamData.jsx [hunk 1]
```diff
@@ -1,453 +1,224 @@
...
+                        bootstrapIcon: 'bi bi-linkedin',
                     },
                 ]
             },
             {
-                image: Image,
-                name: 'Alexander Warren',
+                image: Sae,
+                name: 'Sae Nuruki',
                 jobTitle: 'Founder',
```
Comment (saenuruki) [inline components/Team/TeamData.jsx:34]: @Tch4lla 
Sorry, please update like this.
`Founder・Hackathon Enthusiast`
If there is no space to fill within 1 line, I'll think it again.

File: components/Team/TeamData.jsx [hunk 2]
```diff
@@ -1,453 +1,224 @@
...
                     },
                 ]
             },
             {
-                image: Image,
-                name: 'Alexander Warren',
-                jobTitle: 'Founder',
+                image: Taku,
+                name: 'Takuya Nishikawa',
+                jobTitle: 'Community Manager',
```
Comment (saenuruki) [inline components/Team/TeamData.jsx:91]: He is not active in Discord, but actually he is a founder of this community too.
And he also want to use the title.
Please update it like the following. (I think we don't need to name "Co-Founder")
`Founder・Career Cousultant`

File: components/Team/TeamData.jsx [hunk 3]
```diff
@@ -1,453 +1,224 @@
...
+            description: "Our team's core members hail from diverse backgrounds around the world. We are committed to fostering inclusivity and warmly embrace engineers of all levels of experience.",
             ViewAllMembersUrl: "#"
         },
         members: [
             {
-                image: Image,
-                name: 'Alexander Warren',
-                jobTitle: 'Founder',
+                image: Kent,
+                name: 'Kent',
```
Comment (saenuruki) [inline components/Team/TeamData.jsx:14]: He is `Kenta Uehara`.

**Comment by Tch4lla** [comment]
Thank you for the clarification @saenuruki. I have made those changes

**Review by saenuruki — APPROVED** [review]

**Pull request merged**